app/assets/javascripts/app/discussions/index/discussions.js

```diff
@@ -15,6 +15,7 @@
       return Notification.getAll().length;
     };
 
+    User.findAll();
```

Comment:
I'd move this to the router's root controller, in order to have this triggered regardless of the first route we get in the first